Yoast/wordpress-seo
# Canonical presentations

## Summary


This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Requires the number of pages to be saved in the indexable. So this will not work properly for pages that have not been saved again.
* Added presentation-instance-helpers class so the helpers for the indexable-presentation don't have to be defined in every presentation-instance-builder.

## Test instructions

This PR can be tested by following these steps:

* Create a post with multiple pages. You can achieve this by using the page break block.
* Publish to post.
* On the front end verify the following:
  * Correct canonical.
  * Rel next (except on the last page)
  * Rel prev (except on the first page)
* Make sure the links properly link tp the correct pages.

## UI changes

## Documentation

## Quality assurance


Fixes #
